src/StrawberryfieldHydroponicsService.php

```diff
@@ -181,64 +297,116 @@ public function run() {
 
         $pid = exec(
           sprintf("%s > /dev/null 2>&1 & echo $!", $cmd)
+          //sprintf("%s > /dev/null & echo $!", $cmd)
```

Comment:
No need to keep the commented part. Can be removed

Reply:
done